src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -224,9 +224,8 @@ void CommandRequestImpl::onTimeOut() {
                                             function_id(),
                                             correlation_id(),
                                             mobile_api::Result::GENERIC_ERROR);
-
+  (*response)[strings::msg_params][strings::info] = info;
   AddTimeOutComponentInfoToMessage(response);
```

Comment:
@dcherniev eventually, it does go there, but there's no "info" field afterwards, so I had to add it manually.